docs/sql-programming-guide.md

```diff
@@ -2073,6 +2073,8 @@ options.
      --conf spark.sql.hive.thriftServer.singleSession=true \
      ...
    {% endhighlight %}
+ - Since 1.6.1, withColumn method in sparkR supports adding a new column to or replacing existing columns
```

Comment:
maybe we want to put this in the R migration guide session instead of SQL? or both?